app/models/govbox/message.rb

```diff
@@ -27,6 +27,8 @@ class Govbox::Message < ApplicationRecord
 
   def self.create_message_with_thread!(govbox_message)
     message = nil
+    message_draft = Upvs::MessageDraft.where(uuid: govbox_message.message_id).joins(:thread).where(thread: { box_id: govbox_message.box.id }).take
+    tags_to_migrate = message_draft&.tags_to_migrate_to_message
```

Comment:
Tato migracia tagov sa mi zda taka "rozbita". Nevieme to nejako zabalit do metody, ktora tu urobi v ramci toho advisory_lock?

Reply:
Hej, nebolo to dobre. Ten deferred constraint pomohol a zbavili sme sa tych skaredych casti podla mna.